src/pages/changelog/index.md

```diff
@@ -11,6 +11,9 @@ contributors:
   - https://github.com/padmkris123
 ---
 # Changelog
+## InDesign v19.2
+#### Updated
+- **[BREAKING CHANGE]** UXP logs and console.log [paths](/scripts/tutorials/debug/) have changed.
```

Comment:
Quick question, are these changes only pertaining to scripts or are they applicable to plugins as well?

Reply:
Both.